check_os.py

```diff
@@ -1,9 +1,18 @@
+import Tkinter
```

Comment:
Could you import `Tkinter` in the function itself instead of globally? Otherwise we are stuck with the `Tkinter` module loaded the rest of the application's lifetime, even if we will never need it.